maps/tether/tether-01-surface1.dmm

```diff
@@ -13052,6 +13052,12 @@
 /obj/machinery/door/firedoor/glass/hidden/steel{
 	dir = 1
 	},
+/obj/structure/extinguisher_cabinet{
+	dir = 1;
+	icon_state = "extinguisher_closed";
+	pixel_y = -32;
+	tag = "icon-extinguisher_closed (NORTH)"
```

Comment:
Do me a favor and scrape these tags out. Remove the semicolon from the line above after.